src/app/pages/auth/login/LoginView.js

```diff
@@ -14,13 +16,28 @@ LoginView.defaultProps = {
   valid: true,
 }
 
+// ussage with variables
+const title = gettext('Login')
+
 export default function LoginView({ handleSubmit, submitting, valid }) {
+  useEffect(() => {
+    notification.success({
+      description: gettext('message description'),
```

Comment:
`gettext` will be executed in another context here

Reply:
this is not a case... Notification will render React Component